modules/discourse/iam.tf

```diff
@@ -0,0 +1,51 @@
+variable "iam-assume-role-policy" {}
+
+data "aws_iam_policy_document" "policy-document" {
+    statement {
+        effect  = "Allow"
+        actions = [
+            "s3:*",
+        ]
+
+        resources = [
+            "${aws_s3_bucket.discourse-content.arn}",
+        ]
+    }
+
+    statement {
+        effect  = "Allow"
+        actions = [
+            "s3:*",
+        ]
+
+        resources = [
+            "${aws_s3_bucket.discourse-content.arn}/*",
+        ]
+    }
+
+    statement {
+        effect  = "Allow"
+        actions = [
+            "ses:SendRawEmail",
+        ]
+
+        resources = [
+            "*",
```

Comment:
Shouldn't this be restricted to specific SES resources?

Reply:
It should, but it's current policy is `*`. I'd like to avoid changing actual policies in this PR so we can separate permissions issues from proxy issues.